examples/simple_trigger.py

```diff
@@ -3,7 +3,7 @@
 from airflow.utils.dates import days_ago
 
 
-def generate_dag_run():
+def generate_dag_run(**kwargs):
```

Comment:
Unused variable looks really strange. As for me, it looks like some bug in the code.

Reply:
Agree, but this was necessary when passing the context always. Resolved by the updates.